gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResClassTask.kt

```diff
@@ -33,6 +35,13 @@ abstract class GenerateResClassTask : DefaultTask() {
             val rootResDir = resDir.get().asFile
             logger.info("Generate resources for $rootResDir")
 
+            rootResDir.walkTopDown()
+                .filter { file -> file.name == DS_STORE_FILE }
+                .forEach { file ->
+                    logger.info("Delete $DS_STORE_FILE: ${file.path}")
+                    file.delete()
```

Comment:
I am not sure that it is ok to delete files that we don't create. Mat we just ignore hidden files?